docs/test-specifications/contract-service/ContractCreateTransaction.md

```diff
@@ -0,0 +1,323 @@
+| 2       | Create a contract with no admin key and valid initial balance      | initialBalance="1000"              | Transaction succeeds; `ContractInfo.balance` reflects change. | N           |
+| 3       | Create a contract with an admin key and with zero initial balance  | initialBalance="0"                 | Succeeds, balance remains zero.                               | N           |
+| 4       | Create a contract with no admin key and with zero initial balance  | initialBalance="0"                 | Succeeds, balance remains zero.                               | N           |
+| 5       | Create a contract with an admin key and negative balance           | initialBalance="-100"              | Fails with `CONTRACT_NEGATIVE_VALUE`.                         | N           |
+| 6       | Create a contract with no admin key and negative balance           | initialBalance="-100"              | Fails with `CONTRACT_NEGATIVE_VALUE`.                         | N           |
+| 7       | Create a contract with an admin key and creater than payer balance | initialBalance=\<PAYER\_BALANCE>+1 | Fails with `INSUFFICIENT_PAYER_BALANCE`.                      | N           |
+| 8       | Create a contract with no admin key and creater than payer balance | initialBalance=\<PAYER\_BALANCE>+1 | Fails with `INSUFFICIENT_PAYER_BALANCE`.                      | N           |
+| 9       | Create contract with admin key and initial balance = int64 min       | initialBalance="-9223372036854775808"     | Fails with `CONTRACT_NEGATIVE_VALUE                  | N           |
+| 10      | Create contract without admin key and initial balance = int64 min    | initialBalance="-9223372036854775808"     | Fails with `CONTRACT_NEGATIVE_VALUE                  | N           |
+| 11      | Create contract with admin key and initial balance = int64 min + 1   | initialBalance="-9223372036854775807"     | Fails with `CONTRACT_NEGATIVE_VALUE                  | N           |
+| 12      | Create contract without admin key and initial balance = int64 min + 1| initialBalance="-9223372036854775807"     | Fails with `CONTRACT_NEGATIVE_VALUE                  | N           |
+| 13       | Create contract with admin key and initial balance = int64 max - 1   | initialBalance="9223372036854775806"      | Fails with `INSUFFICIENT_PAYER_BALANCE`.            | N           |
+| 14      | Create contract without admin key and initial balance = int64 max - 1| initialBalance="9223372036854775806"      | Fails with `INSUFFICIENT_PAYER_BALANCE`.             | N           |
+| 15      | Create contract with admin key and initial balance = int64 max       | initialBalance="9223372036854775807"      | Fails with `INSUFFICIENT_PAYER_BALANCE`.             | N           |
+| 16      | Create contract without admin key and initial balance = int64 max    | initialBalance="9223372036854775807"      | Fails with `INSUFFICIENT_PAYER_BALANCE`.             | N           |
```

Comment:
Tests that create a contract with an admin key don't provide an admin key input

Reply:
This is the same for `Auto-Renew Period`, `Memo`, `Staked Account/Node ID`, and `Decline Staking Rewards`.